### PR TITLE
electron-37 : fix for installer pane

### PR DESCRIPTION
**Pull request by VishwasShashidhar**
currently, once an user clicks on "continue" without changing any settings, the plugin is not capturing the details because the correct listener was not being used. 

Now, we use a listener "willExitPane" which allows us to capture and process the data entered by the user.

Please check the code in "MyInstallerPane.m" file to see these changes.

Apart from this, we've also added a plist file for auto-launch in the postinstall script.

### CODE REVIEW

**Comment by VishwasShashidhar** [comment]
@lneir Please review.

File: tests/activityDetection.test.js [hunk 1]
```diff
@@ -1,20 +1,20 @@
 // const activityDetection = require('../js/activityDetection/activityDetection.js');
 // const electron = require('./__mocks__/electron');
 
-xdescribe('Tests for Activity Detection', function() {
+xdescribe('Tests for Activity Detection', function () {
```
Comment (lneir) [inline tests/activityDetection.test.js]: why add space here?  this is not part of our linting rules???

also, other changes not related to the PR should be done separately.  thanks.

**Comment by lneir** [comment]
changes look fine but looks like you need to rebase PR.
also linting changes for other files - not related to this PR - should be done separately.

**Comment by VishwasShashidhar** [comment]
Closing the PR to raise a new one with the correct changes

**Comment by VishwasShashidhar** [comment]
raised a new PR #115

**Pull request closed without merging**